### DISCUSSION TITLE
Integrate with Tubecast to provide Netflix casting support?

### DISCUSSION THREAD

**Discussion by johnhamelink** [CastagnaIT/plugin.video.netflix#1274] [Ideas]
Tubecast is a Kodi addon which allows the YouTube mobile app to "cast" content to the YouTube addon, also in Kodi. It'd be great to provide the same integration with the Netflix addon, so that one could use the existing Netflix app in the same way.

**Comment by CastagnaIT** [comment]
Tubecast/Youtube use a deprecated/old DIAL protocol, not compatible,
i have developed in the past a new cast app, that use the last version of DIAL protocol, here:
https://github.com/CastagnaIT/script.appcast

this is done as plugin mode, where can be add the support for more Kodi app at same time (youtube/netflix/etc...) without fragmenting services that use same ports, but need to be done the implementation for each app, i didn't have time to finish/improve it but i tested locally and works.

if I find the old implementation files I used for testing I will put them on github

Netflix "cast" works differently than YouTube, then must be studied and maybe also some reverse engineering is needed,
the entry point works, but OFC everything else is missing for the data exchange.

I'm not willing to go any further, the project is open source, whoever has time can continue working on it

**Reply by johnhamelink** [reply]
Thanks for your thorough reply, that clarifies a lot for me!
